gke-deploy/README.md

```diff
@@ -0,0 +1,45 @@
+# GKE Deploy
+
+** Warning: This builder is experimental and is very likely to change in
+breaking ways at this time. **
+
+This tool deploys an application to a GKE cluster in a way that follows Google's
+recommended best practices.
+
+## gke-deploy vs kubectl
+
+Using `gke-deploy` to deploy an application to GKE differs from kubectl in that a
+`gke-deploy` deployment follows Google's recommended best practices by doing the
+following:
+
+### Prepare Step
+
+1. `gke-deploy` modifies a set of Kubernetes resource YAML configs to use a
+   container image's digest instead of a tag and adds [recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#applications-and-instances-of-applications).
+
+### Apply Step
+
+2. `gke-deploy` gets authorized to access a GKE cluster.
```

Comment:
Is it intentional that this numbering continues from the above section?

Reply:
Fixed.